src/probability/independence.lean

```diff
@@ -342,4 +342,30 @@ lemma indep_sets.indep_set_of_mem (hs : s ∈ S) (ht : t ∈ T) (hs_meas : measu
 
 end indep_set
 
+section indep_fun
+
+variables {α β β' γ γ' : Type*} [measurable_space α] [measurable_space β] [measurable_space β']
+variables [measurable_space γ] [measurable_space γ'] {μ : measure α}
+
+lemma indep_fun_of_indep_fun_of_ae_eq {f g f' g' : α → β} (hfg : indep_fun f g μ)
```

Comment:
I think we should shorten the two names by removing the `of_indep_fun`, which should be obvious. And we could use dot notation:
```suggestion
lemma indep_fun.ae_eq {f g f' g' : α → β} (hfg : indep_fun f g μ)
```
That name fits well with others in the library, like `uniform_integrable.ae_eq`.